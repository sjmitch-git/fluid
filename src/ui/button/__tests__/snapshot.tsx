import { render } from '@testing-library/react'
import { Button } from '..'

it('renders Button unchanged', () => {
	const { container } = render(<Button>Click!</Button>)

	expect(container).toMatchSnapshot()
})
