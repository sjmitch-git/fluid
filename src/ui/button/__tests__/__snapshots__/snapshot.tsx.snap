// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Button unchanged 1`] = `
<div>
  <button
    class="button group flex gap-1 items-center justify-center disabled:opacity-50 disabled:grayscale shadow hover:shadow-lg scale-100 hover:scale-105 hover:disabled:scale-100 disabled:hover:shadow active:ring active:scale-100 focus:ring focus:ring-accent focus-visible:ring focus-within:ring font-semibold uppercase tracking-widest text-base p-3 bg-primary text-white  "
    data-testid="button"
    role="button"
    title="Click"
    type="button"
  >
    Click!
  </button>
</div>
`;
